src/transformers/expect-js.test.js

```diff
@@ -0,0 +1,291 @@
+      expect(stuff).toBeFalsy();
+
+      expect(stuff).toBe('message');
+      expect(stuff).toBe('message');
+
+      expect(stuff).toEqual('message');
+
+      expect(stuff).toContain('message');
+
+      expect(stuff).toHaveLength(1);
+
+      expect(stuff).toHaveProperty('message');
+      expect(stuff).toHaveProperty('message');
+
+      expect(stuff).toMatch('message');
+    });
+    `
+);
+
+testChanged(
+    'does not map non expect.js matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      bob(stuff).to.be.ok();
+      expect(stuff).to.be.bananas();
+    });
+    `,
+    `
+    test(() => {
+      bob(stuff).to.be.ok();
+      expect(stuff).to.be.bananas();
+    });
+    `
+);
+
+testChanged(
+    'maps expect not matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.not.be('message');
+      expect(stuff).to.not.equal('message');
+
+      expect(stuff).to.not.eql('message');
+
+      expect(stuff).to.not.contain('message');
+
+      expect(stuff).to.not.have.length(1);
+
+      expect(stuff).to.not.have.property('message');
+
+      expect(stuff).to.not.match('message');
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff).not.toBe('message');
+      expect(stuff).not.toBe('message');
+
+      expect(stuff).not.toEqual('message');
+
+      expect(stuff).not.toContain('message');
+
+      expect(stuff).not.toHaveLength(1);
+
+      expect(stuff).not.toHaveProperty('message');
+
+      expect(stuff).not.toMatch('message');
+    });
+    `
+);
+
+testChanged(
+    'maps expect number matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.be.lessThan(42);
+      expect(stuff).to.be.below(42);
+      expect(stuff).to.be.greaterThan(42);
+      expect(stuff).to.be.above(42);
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff).toBeLessThan(42);
+      expect(stuff).toBeLessThan(42);
+      expect(stuff).toBeGreaterThan(42);
+      expect(stuff).toBeGreaterThan(42);
+    });
+    `
+);
+
+testChanged(
+    'maps expect throw matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.throwError(Error);
+      expect(stuff).to.throwError();
+      expect(stuff).to.throw(Error);
+      expect(stuff).to.throw();
+      expect(fn).to.throwException(function (e) {
+        expect(e).to.be.a(SyntaxError);
+      });
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff).toThrowError(Error);
+      expect(stuff).toThrow();
+      expect(stuff).toThrowError(Error);
+      expect(stuff).toThrow();
+      try {
+        fn();
+        fail();
+      } catch (e) {
+        expect(e).toBeInstanceOf(SyntaxError);
+      };
+    });
+    `
+);
+
+testChanged(
+    'maps expect fail matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect().fail('message');
+    });
+    `,
+    `
+    test(() => {
+      fail('message');
```

Comment:
I tested and `fail` does work as expected and I *think* it's supported [here](https://github.com/facebook/jest/blob/master/packages/jest-jasmine2/src/jasmine/jasmine_light.js#L115) but perhaps the best thing to do is just throw a new Error instead of using this.